modules/tinymce/src/core/test/ts/browser/html/SchemaTest.ts

```diff
@@ -566,6 +576,35 @@ describe('browser.tinymce.core.html.SchemaTest', () => {
 
       assert.hasAnyKeys(schema.getNonEmptyElements(), [ 'foo-bar', 'FOO-BAR', 'bar-foo', 'BAR-FOO' ]);
     });
+
+    it('TINY-10139: Check block elements', () => {
+      const schema = Schema({
+        custom_elements: 'foo,bar'
+      });
+      checkElement('p', (el) => schema.isBlock(SugarNode.name(el)), true);
+      checkElement('h1', (el) => schema.isBlock(SugarNode.name(el)), true);
+      checkElement('table', (el) => schema.isBlock(SugarNode.name(el)), true);
+      checkElement('span', (el) => schema.isBlock(SugarNode.name(el)), false);
+      checkElement('b', (el) => schema.isBlock(SugarNode.name(el)), false);
+      checkText((el) => schema.isBlock(SugarNode.name(el)));
```

Comment:
You can move these to separate `context('isBlock')`

Reply:
I moved `isBlock` and `isInline` inside a new content called: `check elements`, I think it make sense to keep these checks inside the same context